examples/chip-tool/templates/tests/partials/test_cluster.zapt

```diff
@@ -72,6 +72,11 @@ private:
     CHIP_ERROR DoTestStep(uint16_t testIndex) override
     {
         using namespace chip::app::Clusters;
+        // Allow yaml to access the current commissioner node id.
```

Comment:
nit, might be nice to add in comment that getting this value is done before every TestStep because you don't know when this value may have changed

Reply:
added